di-ipv-cimit-stub/lambdas/put-contra-indicators/src/main/java/uk/gov/di/ipv/core/putcontraindicators/service/ContraIndicatorsService.java

```diff
@@ -117,40 +117,35 @@ private ContraIndicatorEvidenceDto parseContraIndicatorEvidence(SignedJWT signed
     }
 
     private List<CimitStubItem> mapToContraIndications(
-            String userId, ContraIndicatorEvidenceDto contraIndicatorEvidenceDto) {
+            String userId,
+            ContraIndicatorEvidenceDto contraIndicatorEvidenceDto,
+            Instant issuanceDate) {
         if (contraIndicatorEvidenceDto.getCi().isEmpty()) {
```

Comment:
We don't need this exception throw....just log for info.
Process VC which got ci block.